react/features/chat/components/native/ChatInputBar.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import React, { Component } from 'react';
+import { TextInput, View } from 'react-native';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * Callback to invoke on message send.
+     */
+    onSend: Function
+};
+
+type State = {
+
+    /**
+     * The value of the input field.
+     */
+    message: string
+};
+
+/**
+ * Implements the chat input bar with text field and action(s).
+ */
+export default class ChatInputBar extends Component<Props, State> {
```

Comment:
Let's call this `ChatInputField`, it's not a bar and there are references to "field" alesewhere here.